_ingest-pipelines/processors/fail.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: default
+title: Fail
+parent: Ingest processors
+nav_order: 100
+---
+
+# Fail processor
+
+The `fail` processor can be useful for performing data transformations and enrichment during the indexing process. The primary use case for the `fail` processor is to fail an indexing operation when certain conditions are met.
+
+The following is the syntax for the `fail` processor:
+
+```json
+"fail": { 
+  "if": "ctx.foo == 'bar'", 
+  "message": "Custom error message" 
+  }
+```
+{% include copy-curl.html %}
+
+## Configuration parameters
+
+The following table lists the required and optional parameters for the `fail` processor.
+
+Parameter | Required/Optional | Description |
```

Comment:
Nit: I would just say "Required" in the second column, since its a little cleaner on the table, though it looks like the other processor docs follow the same convention here.